### PR TITLE
Minimal changes to pass SonarQube Code scan

### PR DESCRIPTION
**Pull request by msagolj**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | #813 
| Patch: Bug Fix?          | no
| Minor: New Feature?      | no
| Major: Breaking Change?  | no
| Tests Added + Pass?      | -
| Documentation Provided   | no
| Any Dependency Changes?  | no
| License                  | Apache License, Version 2.0

Applied code changes so that the core bundle passes the code scanning step in Adobe AMS/Cloud pipeline.

### CODE REVIEW

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public static String getURL(@NotNull SlingHttpServletRequest request, @NotNull P
     @NotNull
     public static String getURL(@NotNull SlingHttpServletRequest request, @NotNull Page page) {
         String vanityURL = page.getVanityUrl();
-        return StringUtils.isEmpty(vanityURL) ? request.getContextPath() + page.getPath() + ".html" : request.getContextPath() + vanityURL;
+        return StringUtils.isEmpty(vanityURL) ? ( request.getContextPath() + page.getPath() + ".html" ): (request.getContextPath() + vanityURL);
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java]: I'd remove the space after "(" and before ")".

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java [hunk 2]
```diff
@@ -217,8 +217,8 @@ protected void initModel() {
                     smartImages[index] = baseResourcePath + DOT +
                         selector + DOT + jpegQuality + DOT + width + DOT + extension +
                         (inTemplate ? Text.escapePath(templateRelativePath) : "") +
-                        (lastModifiedDate > 0 ? "/" + lastModifiedDate +
-                        (StringUtils.isNotBlank(imageName) ? "/" + imageName : "") + DOT + extension : "");
+                        (lastModifiedDate > 0 ? ("/" + lastModifiedDate +
+                        (StringUtils.isNotBlank(imageName) ? ("/" + imageName) : "") + DOT + extension ): "");
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java]: I'd remove the space after "(" and before ")".
This applies to all the changes.

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java (hunk 1 above)
Comment (msagolj) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java]: removed spaces in later commit.

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java [hunk 3]
```diff
@@ -162,7 +162,6 @@ private long getResourceSize (Resource resource) {
                     }
                 }
             } catch (RepositoryException e) {
-                LOG.info ("Cannot determine size of binary at {}", resource.getPath(),e);
```
Comment (bpauli) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java]: We should log an error here

Comment (msagolj) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java]: Done, see follow up commit

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ExperienceFragmentImpl.java [hunk 4]
```diff
@@ -204,7 +203,6 @@ private String getLiveCopyPath(Resource resource) {
                 }
             }
         } catch (WCMException e) {
-            LOGGER.debug("Unable to get the live copy: {}", e.getMessage());
```
[the remark this answers is not in the document]
Comment (msagolj) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ExperienceFragmentImpl.java]: Done, see follow up commit.

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ExperienceFragmentImpl.java [hunk 5]
```diff
@@ -180,7 +180,6 @@ private String getBlueprintPath(Resource resource) {
                 }
             }
         } catch (WCMException e) {
-            LOGGER.debug("Unable to get the blueprint: {}", e.getMessage());
```
[the remark this answers is not in the document]
Comment (msagolj) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ExperienceFragmentImpl.java]: Done , see follow up commit.

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/DownloadImpl.java [hunk 6]
```diff
@@ -282,6 +282,7 @@ public String getExtension() {
         return extension;
     }
 
+    @SuppressWarnings("squid:S00112")
```
[the remark this answers is not in the document]
Comment (msagolj) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/DownloadImpl.java]: Done, see follow up commit.

**Pull request merged**